doc/quickstart/configure.rst

```diff
@@ -289,6 +292,48 @@ related to CMOR table settings available:
   to get the name of the file containing the ``mip`` table.
   Defaults to the value provided in ``cmor_type``.
 
+.. _configure_native_models:
+  
+Configuring native models and observation data sets
+----------------------------------------------------
+
+ESMValTool can take full advantage of the ability to configure
+ESMValCore for handling native model output formats and specific
+observation data sets without preliminary reformating. You can choose
+to host this new data source either under a dedicated project or under
+project ``native6``; when choosing the latter, such a configuration
+involves the following steps :
+
+  - allowing for ESMValTool to locate the data files :
+
+    - entry ``native6`` of ``config-developer.yml`` should be
+      complemented with sub-entries for ``input_dir`` and ``input_file``
+      that goes under a new key representing the
+      data organization (such as ``NEW_MODEL``), and these sub-entries can
```

Comment:
```suggestion
      data organization (such as ``INSTITUTE``), and these sub-entries can
```
these names are usually institute names, not model names, because they refer to the data storage organization used at a particular institute.